com.unity.render-pipelines.universal/Runtime/RenderTargetBufferSystem.cs

```diff
@@ -0,0 +1,74 @@
+using System;
```

Comment:
The working of this class is not how we want it to be. I made it like this to quickly tests if we can achieve a swapbuffer system without RTHandles. I'm working on a better solution.

If you have any suggestions on how we should handle this class. Please feel free to suggest.